### ISSUE TITLE
Release on clojars?

### ISSUE THREAD

**Issue by metasoarous** [appliedsciencestudio/darkstar#3]
Would you please publish this on Clojars? I'm switching Oz over to `deps.edn`, so I'm able to use the darkstar git coordinates from my project, but unless I package Oz as an uberjar (which has caused other issues in the past), my understanding is that things might break for folks using `lein` because it won't be able to find the package.

Thanks in advance!

**Comment by jackrusher** [comment]
I intend to do so, but I do want to add some tests and (hopefully) get more user feedback before I cut an official version. Speaking of which, how are things going for you in using the library? :)

**Comment by cnuernber** [comment]
@metasoarous - tech.viz is on clojars.  It includes the this library.

**Comment by metasoarous** [comment]
Yeah, I went ahead and threw up `metasoarous/darkstar` on clojars for now. Will use that for now. Thanks!

**Comment by harold** [comment]
This library rocks, please release it on clojars. (^:

**Comment by mars0i** [comment]
Why not put up a non-official version on Clojars?  When there's an official version, that will supercede it.  I can use the metasoarous version, though.

**Comment by jackrusher** [comment]
I don't currently have the time and headspace to deal with the new requirements at Clojars:
https://github.com/clojars/administration/issues/244

**Comment by mars0i** [comment]
I didn't know about all of that.  Thanks for explaining.  I'm much more comfortable with Leingingen than deps.edn at present, and wanted to try Darkstar in an existing project that uses Leiningen.  Not your problem, or your fault.  Clojars is making it harder.

**Comment by jackrusher** [comment]
In the meantime you can use the version @metasoarous published on Clojars, `metasoarous/darkstar`.

**Comment by mars0i** [comment]
Yes, thanks.  I just got that running.
